web/projects/ui/src/app/services/patch-db/patch-db.factory.ts

```diff
@@ -27,36 +27,34 @@ export function sourceFactory(
   // defer() needed to avoid circular dependency with ApiService, since PatchDB is needed there
   return defer(() => {
     const api = injector.get(ApiService)
-    const authService = injector.get(AuthService)
+    const auth = injector.get(AuthService)
+    const state = injector.get(StateService)
     const connectionService = injector.get(ConnectionService)
-    const configService = injector.get(ConfigService)
     const bootstrapper = injector.get(LocalStorageBootstrap)
-    const isTor = configService.isTor()
-    const timeout = isTor ? 16000 : 4000
 
-    return from(api.subscribeToPatchDB({})).pipe(
-      switchMap(({ dump, guid }) => {
-        const websocket$ = api.openWebsocket$<Revision>(guid, {}).pipe(
-          bufferTime(250),
-          filter(revisions => !!revisions.length),
-          // @TODO catch error should defer back to the global poll for state
-          catchError((_, watch$) => {
-            connectionService.websocketConnected$.next(false)
-
-            return interval(timeout).pipe(
-              switchMap(() =>
-                from(api.getState()).pipe(catchError(() => EMPTY)),
-              ),
-              take(1),
-              switchMap(() => watch$),
-            )
-          }),
-          tap(() => connectionService.websocketConnected$.next(true)),
-          startWith([dump]),
-        )
+    return defer(() => from(api.subscribeToPatchDB({}))).pipe(
+      switchMap(({ dump, guid }) =>
+        auth.isVerified$.pipe(
+          switchMap(verified =>
+            verified
+              ? api.openWebsocket$<Revision>(guid, {}).pipe(
+                  bufferTime(250),
+                  filter(revisions => !!revisions.length),
+                  tap(() => connectionService.websocketConnected$.next(true)),
```

Comment:
With global state monitoring, I don't think the websocketConnected$ observable is necessary anymore.